tests/test_sourceMatch.py

```diff
@@ -181,6 +181,7 @@ def testPhotometricCalib(self):
             s.setDec(dec * afwGeom.degrees)
             s.set(self.table.getPsfFluxKey(), psfMags[band])
 
+        ifd.close()
```

Comment:
I'm happy to do that. The little Robert voice in my head was wondering if the big indenting diff was worth it.